internal/pkg/manifest/rd_web_svc.go

```diff
@@ -48,9 +48,9 @@ type RequestDrivenWebServiceProps struct {
 
 // AppRunnerInstanceConfig contains the instance configuration properties for an App Runner service.
 type AppRunnerInstanceConfig struct {
-	CPU      *int    `yaml:"cpu"`
-	Memory   *int    `yaml:"memory"`
-	Platform *string `yaml:"platform,omitempty"`
+	CPU      *int                 `yaml:"cpu"`
+	Memory   *int                 `yaml:"memory"`
+	Platform PlatformArgsOrString `yaml:"platform,omitempty"`
```

Comment:
Did we already ship this in v1.9?

I think it'd be best for app runner if we always set the platform under the hood to `linux/x86_64` and not provide this as a configurable option.

Reply:
Okay, should we surface the field at all for App Runner?